### PR TITLE
Move test pages off glitch.

### PR DESCRIPTION
**Pull request by sammacbeth**
## Description:
Replace occurrences of `privacy-test-pages.glitch.me` with `privacy-test-pages.site`.

### CODE REVIEW

**Review by KeaganJarvis — COMMENTED** [review]
This looks good from my end but will wait for tests to finish and for an approval from one of the requested reviewers 👍

**Review by kzar — APPROVED** [review]

**Pull request merged**
